services/secret-service/server/pom.xml

```diff
@@ -47,7 +47,7 @@
         <dependency>
             <groupId>org.apache.airavata</groupId>
             <artifactId>airavata-credential-store-stubs</artifactId>
-            <version>0.19-SNAPSHOT</version>
+            <version>0.17</version>
```

Comment:
Why?

Reply:
This is now reverted back to its original version of 0.19-SNAPSHOT.